src/test/java/Domain/Controllers/TeamControllerTest.java

```diff
@@ -73,11 +76,11 @@ public void AddTeamOwner2UTest() throws Exception{
         EntityManager.getInstance().addUser(teamOwnerUser);
         hapoelBash.addTeamOwner(teamOwnerUser.getRole(RoleTypes.TEAM_OWNER));
         try{
-            TeamController.addTeamOwner("oranSh", hapoelBash, (TeamOwner)teamOwnerUser.getRole(RoleTypes.TEAM_OWNER));
+            TeamController.addTeamOwner("dfds", hapoelBash, (TeamOwner)teamOwnerUser.getRole(RoleTypes.TEAM_OWNER));
```

Comment:
When you are expecting a test to fail don't forget to add "assert.fail()" at the end of "try", to make the test both fail and show the message you want. Otherwise the test will pass if the statements in "try" won't throw exceptions.